.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,41 @@
+name: Release metrics adapter chart
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
```

Comment:
nice! I did not know there was actually a name for that issue.

Reply:
Neither do I, is a copy and paste from the actual `helm-chart` repo where I learn about it.